Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -120,7 +119,9 @@ private Furniture(Furniture other)
 
         Parameters = new Parameter(other.Parameters);
         Jobs = new BuildableJobs(this, other.Jobs);
-        workshop = other.workshop; // don't need to clone here, as all are prototype things (not changing)
+
+        // don't need to clone here, as all are prototype things (not changing)
```

Comment:
We shouldn't assume this. Once we make a component that will have params that change, then this will not be true.